cmake/sca/eclair/ECL/reports.ecl

```diff
@@ -37,21 +37,33 @@ loading()
 map_strings("load_ecb", dir_entries(data_dir))
 loaded()
 
-if(string_equal(or(getenv("TEXTUAL_REPORTS"),"false"),"true")
-   eval_file("report__textual.ecl"))
-
-
-# Output report summaries in ODT format.
-#-summary_odt=join_paths(output_dir,"odt")
-# Output report summaries in ODT format.
-#-summary_doc=join_paths(output_dir,"doc")
-# Output report summaries in pure text format.
-#-summary_txt=join_paths(output_dir,"txt")
-# Output full report in pure text format.
-#-full_txt=join_paths(output_dir,"txt")
-# Output metrics for use with spreadsheet applications (if enabled).
-#-metrics_tab=join_paths(output_dir,"metrics")
-# Output reports for use with spreadsheet applications
-#-reports_tab=join_paths(output_dir,"reports")
+if(string_equal(or(getenv("ECLAIR_metrics_tab"),"false"),"true")
```

Comment:
Using env variables here might be very convenient for someone who wrote the integration but it seems very hidden and undocumented for everyone else.

Also picking up environment variables during build can lead to hard to reproduce builds.
We should probably consider to allow SCA tools to provide a `Kconfig` file.

Reply:
They are not undocumented, you can find the documentation in sca/eclair.rst.
In any case, I've got the point. I'm not really familiar with Kconfig, I'll give a look
at it.

Reply:
> They are not undocumented, you can find the documentation in sca/eclair.rst.

Try read my comment again.
I'm not talking about the settings themselves, but the fact they are being picked up from environment.
That behavior is hidden and undocumented.

https://github.com/zephyrproject-rtos/zephyr/blob/6b67643eb9cdc111f367fa18b60e9e695be11efb/doc/develop/sca/eclair.rst?plain=1#L66-L67
https://github.com/zephyrproject-rtos/zephyr/blob/6b67643eb9cdc111f367fa18b60e9e695be11efb/doc/develop/sca/eclair.rst?plain=1#L93

Reply:
That is true allowing SCA tools to provide a Kconfig would be good thing to make it clearer what can be configured for the tool itself and also would document the configuration and behavior of the sca tooling better.

But is there actual a better way to do something like this with SCA and cmake? Because I couldn't find a better way to do this right now.